topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -241,14 +241,14 @@ To determine where DNA fragments originated in the genome, the sequenced reads m
 >        - *"Reference genome with or without an annotation"*: `use genome reference with builtin gene-model`
 >            - *"Select reference genome"*: `Homo sapiens (hg38+GRCh38.87)`
 >    - *"Count number of reads per gene"*: `No`
->    - *"Would you like to set output parameters (formatting and filtering)?"*: `Yes`
->        - *"Extra SAM attributes to include"*: `All`
->        - *"Include strand field flag XS"*: `Yes -- and reads with inconsistent and/or non-canonical introns are filtered out`
->        - *"Would you like to set additional output parameters (formatting and filtering)?"*: `Yes`
->    - *"Other parameters (seed, alignment, limits and chimeric alignment)"*: `Extended parameter list`
->        - In *"Alignment parameters"*:
->            - *"Use end-to-end read alignments, with no soft-clipping?"*: `Yes`
->        - *"Would you like to set chimeric alignment parameters?"*: `No`
+>    - In *"BAM output format specification"*:
```

Comment:
do we maybe need to annotate the tool version here? GTN can fix tool versions of specific tools

Reply:
(Good suggestion! I should add that to linting)

Reply:
Sorry, I'm not sure what you mean by annotating the tool version.

Reply:
If you could use the new tool linking syntax it would be really nice for users!

We have a [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links), but in short just switch to `{% tool [Your Tool](toolshed.g2.bx.psu.edu/repos/..../your-tool/1.0.0+galaxy0) %}`

Reply:
Ahh I see, thanks a lot!